packages/suite/src/views/wallet/coinmarket/index.tsx

```diff
@@ -17,6 +17,10 @@ export const Wrapper = styled.div<ResponsiveSize>`
     }
 `;
 
+export const FullWidthForm = styled.div`
```

Comment:
```suggestion
export const FullWidthForm = styled.form`
```

otherwise nothing happens on `Enter` key